.github/CODEOWNERS

```diff
@@ -32,9 +32,14 @@
 /docker             @getsentry/releases
 setup.py            @getsentry/releases
 setup.cfg           @getsentry/releases
-# TODO: owners-js-build, owners-build (for travis / gha)
+# TODO: owners-build (for travis / gha)
 requirements*.txt   @getsentry/owners-python-build
 pyproject.toml      @getsentry/owners-python-build
+tsconfig.*          @getsentry/owners-jsbuild
```

Comment:
Can you rename to `owners-js-build`? For consistency.

Reply:
I'll see if I can get @dcramer to do real fast

Reply:
done https://github.com/orgs/getsentry/teams/owners-js-build